scripts/utils/trading_strategy_helpers.js

```diff
@@ -292,7 +289,7 @@ module.exports = function(web3 = web3, artifacts = artifacts) {
     // Sell x ETH for max256 DAI
     // x = max256 / 102
     // priceFormatted = 102000000000000000000
-    price = price.toFixed(18)
+    price = price.toFixed(targetToken.decimals)
```

Comment:
This changes the logic in an incorrect way. I'm not going to adapt this function to avoid conflicts with PR #105, but there should be a test somewhere telling me that changing this value is wrong.